src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuiteLanguage.Table.al

```diff
@@ -0,0 +1,63 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+using System.Globalization;
+
+table 149035 "AIT Test Suite Language"
+{
+    Caption = 'AI Test Suite Language';
```

Comment:
Do we need to persist this table? Can it be a temp table?
Second question - do we need a table at all? Can we get lists?

Reply:
@stkillen I would still calculate it on the fly. I'm not against if you want to keep it in the table, however the less places it is defined in, less headaches.